content/release-10-16-0/announcements-10-16-0.md

```diff
@@ -44,6 +44,10 @@ If you don´t know what is required to switch off SMS TFA, please contact our cu
 
 ### SDK changes
 
+##### New default branding
+
+As of Release 10.17, we will update the default branding for all Cumulocity IoT default applications. Branded applications will not be affected, however, as the new navigator changes from a dark color to a light color, the default navigator font color might not work with your current branding settings. You might end up with a dark font color on a dark background, which might not be readable or accessible by your users. You can review this by opening the branding editor in the Administration application and check what font color you are using in the current version.
```

Comment:
```suggestion
As announced with [release 10.15](/release-10-15-0/announcements-10-15-0), as of Release 10.17, we will update the default branding for all Cumulocity IoT default applications. Branded applications will not be affected, however, as the new navigator changes from a dark color to a light color, the default navigator font color might not work with your current branding settings. You might end up with a dark font color on a dark background, which might not be readable or accessible by your users. You can review this by opening the branding editor in the Administration application and check what font color you are using in the current version.
```

Reply:
We should alsways add this, otherwise people might say, we only announced 1 release in advance :-).